### PR TITLE
Journeys

### PR DESCRIPTION
**Pull request by aaustin**
@zeeshabnc I don't think Brian has github but this is ready to go!

### CODE REVIEW

File: pages/features/journeys.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+- **Draft**: Not live for your users, still editable
+- **Active**: Live for your users, read-only
+- **Archived**: Not live for your users, hidden from default manager view
+
+You can activate a journey directly from the creation flow, or from “Start” in the Actions menus in the Journeys Manager. Editing and Archiving are also accessible from the Actions menus.
+
+{% image src='/img/pages/features/journeys/edit-journeys.png' third center alt='edit journeys' %}
+
+Note that you must `clone` or `pause` a live journey to edit it. They are read only while live.
```
Comment (zeeshabnc) [inline pages/features/journeys.md]: Change pause to Stop?

File: pages/features/journeys.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+## Visualizing Journeys Performance
+
+You can access your Journey’s performance direct from the actions menu in the Journeys Manager. Journeys map to standard Branch analytics tags:
+
+- All Journeys: `feature`: `journeys`
+- Each Journey:	`campaign`:	`<journey name>`
```
Comment (zeeshabnc) [inline pages/features/journeys.md]: campaign: <journey.id>. Reason being, journey names can change and they would mess up the analytics. Hopefully when we do custom analytics for journeys we will display the journey name instead. For now, in GA, we are thinking to add the ability to copy the journey id to clipboard.

File: pages/features/journeys.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+
+You can also access Journeys analytics by selecting the above filters in Source Analytics. To compare all of your Journeys:
+
+1. Filter by feature = journeys
+2. Group by campaign
+
+To compare Branch View segments within one Journey:
+
+1. Filter by feature = journeys
+2. Filter by campaign = Journey name
```
Comment (zeeshabnc) [inline pages/features/journeys.md]: journey.id

File: pages/features/journeys.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+
+If you have custom event tracking set up, you can target users based on events that you define. For instance, you might want to show a Branch View to users who make a purchase, or who add an item to their shopping cart more than three times.
+
+**Referred from site**
+
+You can target a user based on the last touch point before they entered your website. For example, if you want to target users that found you through Google Search, you can select “Referred from site” and fill in `google.com`. Currently, we only support domain names in the Referred from site field.
+
+**Current website url**
+
+You can define which subsets of your website the Journey will appear. For example, maybe you have a page `yoursite.com/settings` and yoursite.com/products/1234`. You could fill in `products` here and only users visiting a URL with that substring present would see the Journey.
```
Comment (zeeshabnc) [inline pages/features/journeys.md]: Is this part of v1? Do the audience filters take care of this (given that we removed the filters on the rules section)?

Comment (aaustin) [inline pages/features/journeys.md]: Will leave this in (as well as other things) just to write the docs for the feature we expect to have over the next week.

**Pull request merged**